Chemistry/CreateHubbardHamiltonian/README.md

```diff
@@ -1,14 +1,8 @@
 ﻿# Hubbard Simulation Sample #
 
-This sample walks through constructing the time-evolution operator for the 1D Hubbard Simulation model. This time-evolution operator is applied to project onto the ground state of Hubbard Hamiltonian using phase estimation.
-
-## Running the Sample ##
-
-Open the `QsharpLibraries.sln` solution in Visual Studio and set the .csproj file in the manifest as the startup project.
-Press Start in Visual Studio to run the sample.
+This sample walks through constructing the Jordan–Wigner representation of the time-evolution operator for the 1D Hubbard simulation model, similarly to the [**HubardHamiltonian**](../../Samples/src/HubbardHamiltonian/) sample, but using the chemistry library to represent the Hubbard model.
 
 ## Manifest ##
 
-- [HubbardSimulation.qs](./HubbardSimulation.qs): Q# code implementing quantum operations for this sample.
```

Comment:
Did this file migrate to the sample linked above?

Reply:
It was never in the sample, it turns out. The CreateHubbardHamiltonian sample concerns the C# part of the chemistry library, rather than the Q# covered by some of the other chemistry samples.